source/_integrations/alarm_control_panel.template.markdown

```diff
@@ -45,7 +45,10 @@ alarm_control_panel:
           target:
             entity_id: alarm_control_panel.real_alarm
           data:
+            # assigning a value to 'code' will disregard any input from the UI, suitable for arming without manually entering the code
             code: !secret alarm_code
+            # assigning the correct template to 'code' will use the UI input, suitable for arming only after entering the correct code
+            #code: '{{code}}'
```

Comment:
This is documentation hidden in the code example. This generally indicates the documentation is not sufficient. I suggest we adjust the documentation around these parameters instead of the example.